atomic_reactor/plugins/pre_add_image_content_manifest.py

```diff
@@ -121,13 +118,19 @@ def __init__(self, workflow, remote_sources=None, destdir=IMAGE_BUILD_INFO_DIR):
 
     @property
     def layer_index(self):
-        if self._layer_index is None:
-            # Default layer index is 1, because base and 'FROM scratch' images
-            #     *always* have 2 layers
-            self._layer_index = 1
-        if not base_image_is_scratch(self.dfp.baseimage):
-            # OSBS2 TBD
-            inspect = imageutil.base_image_inspect()
+        if self._layer_index is not None:
+            return self._layer_index
+
+        # OSBS2 TBD: decide if we need to inspect a specific arch
```

Comment:
we don't because image for all arches should have same amount of layers

Reply:
That's what I'm thinking as well, but technically, nothing in the docker/OCI specification says that all images in a manifest list must be "the same" (and it's not clear to me what exactly that would mean anyway).

It's probably OK to assume that all images in a manifest list have the same number of layers, but I'd rather leave this here to be resolved when we update the content manifest plugin